3scale/porta
# Fixes CSV download in IE11

**What this PR does / why we need it**:

`download` property of anchor (`<a download />`) is not supported in IE11. This PR checks whether the browser is IE and renders a div that calls `msSaveBlob` instead.

**Which issue(s) this PR fixes** 

[[THREESCALE-1318] Can not download Usage CSV in Internet Explorer 11](https://issues.jboss.org/browse/THREESCALE-1318)

**Verification steps** 

* Internet Explorer: In porta, go to any Analytics > Click on Download CSV > Verify CSV is valid
* Any proper browser: In porta, go to any Analytics > Click on Download CSV > Verify CSV is valid

**Style changes**
Disabled:
![csv-0](https://user-images.githubusercontent.com/11672286/48504339-d132fd00-e844-11e8-97be-12ebe6df4119.gif)

Enabled:
![csv-1](https://user-images.githubusercontent.com/11672286/48504447-0d665d80-e845-11e8-83a9-6629208517da.gif)
